cloud_images/centos7/install_prereqs.sh

```diff
@@ -21,7 +21,7 @@ sysctl -w net.ipv6.conf.all.disable_ipv6=1
 sysctl -w net.ipv6.conf.default.disable_ipv6=1
 
 echo ">>> Installing DC/OS dependencies and essential packages"
-yum install --assumeyes --tolerant perl tar xz unzip curl bind-utils net-tools ipset libtool-ltdl rsync
+yum -y --tolerant install perl tar xz unzip curl bind-utils net-tools ipset libtool-ltdl rsync nfs-utils
```

Comment:
Why can't it be a framework's custom executor that's doing the mounting? Maybe RexRay will one day have a featureset like k8s' PersistentVolumes and support NFS. In any case whatever the user uses to mount a NFS volume he's going to require the nfs-utils package.